Makefile

```diff
@@ -113,16 +113,27 @@ srpm: .srpm-done
 
 rpm: .rpm-done
 
-ubuntu-trusty:
-	cp packaging/ubuntu-trusty/ecs.conf ecs.conf
-	tar -czf ./amazon-ecs-init_${VERSION}.orig.tar.gz ecs-init ecs.conf scripts README.md
+ARCH:=$(shell uname -m)
+ifeq (${ARCH},x86_64)
+	AGENT_URL=https://s3.amazonaws.com/amazon-ecs-agent/ecs-agent-v${VERSION}.tar
+else ifeq (${ARCH},aarch64)
+	AGENT_URL=https://s3.amazonaws.com/amazon-ecs-agent/ecs-agent-arm64-v${VERSION}.tar
+endif
+
+download-agent:
+	curl -o /tmp/ecs-agent-v${VERSION}.tar ${AGENT_URL}
+
+deb: download-agent
```

Comment:
You should use a stamp file for dependencies, similar to what's done with the `.rpm-done` file. e.g.
```
.PHONY: deb
deb: .deb-done
.deb-done:
  <rest of recipe>
```

Reply:
will do